frontend/src/assets/scss/layout/_menu.scss

```diff
@@ -0,0 +1,32 @@
+.l-menu{
+  @apply border-r border-gray-200 bg-white py-4 flex flex-col transition-all;
+
+  &__link{
+    @apply w-full flex flex-nowrap items-center border-l-3 h-9 px-5 text-gray-900 text-medium;
```

Comment:
All menu items have a left gray border when they are not selected. I believe this shouldn't exist

![Screenshot 2024-09-18 at 15 28 37](https://github.com/user-attachments/assets/7944ffeb-d0a0-4a92-868f-9c6b9f30bf23)
![Screenshot 2024-09-18 at 15 28 14](https://github.com/user-attachments/assets/503d0d65-c808-47c4-9967-fe16ef496b5e)